project-base/app/src/DataFixtures/Demo/InquiryDataFixture.php

```diff
@@ -394,7 +395,7 @@ public function load(ObjectManager $manager): void
         $inquiryData->createdAt = new DateTimeImmutable('2024-09-02 09:25:41');
         $this->inquiryFacade->create($inquiryData);
 
-        $inquiryData = $this->inquiryDataFactory->create();
+        $inquiryData = $this->inquiryDataFactory->create(Domain::FIRST_DOMAIN_ID);
```

Comment:
I don't find it necessary. 
This may be over-complicated as the inquiries from all domains are displayed in a single grid. 
When we iterate over all domains, we have to handle the duplicities or fake data on other domains. 
TBH, for me it is not worth the time. 
But I'm open to suggestions:)